Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -359,9 +351,8 @@ void FullscreenManager::exitFullscreen(CompletionHandler<void(ExceptionOr<void>)
         // there is a pending task in requestFullscreenForElement(). Cause it to cancel and fire an error
         // by clearing the pending fullscreen element.
         RefPtr exitedFullscreenElement = fullscreenElement();
-        if (!exitedFullscreenElement && m_pendingFullscreenElement) {
+        if (!exitedFullscreenElement) {
             INFO_LOG(identifier, "task - Cancelling pending fullscreen request.");
-            m_pendingFullscreenElement = nullptr;
             m_pendingExitFullscreen = false;
             return completionHandler({ });
         }
```

Comment:
The code comment and the error log are no longer up to date.

You could just say there's no fullscreen element to exit :)